Search times (#4350)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/limits-for-content-search.md

|Maximum body size in index  <br/> |67 million characters  <br/> |The total number of characters in the body of an email message and all its attachments. When an email message is indexed, all text in the body of the message and in all attachments is concatenated into a single string. The maximum size of this string that is indexed is 67 million characters.  <br/> |
|Maximum unique tokens in body  <br/> |1 million  <br/> |As previously explained, tokens are the result of extracting text from content, removing punctuation and spaces, and then dividing it into words (called tokens) that are stored in the index. For example, the phrase  `"cat, mouse, bird, dog, dog"` contains 5 tokens. But only 4 of these are unique tokens. There is a limit of 1 million unique tokens per email message, which helps prevent the index from getting too large with random tokens.  <br/> |
|||

> ## Search Times

Reviewer: ```suggestion
## Search times
```

> Microsoft collects performance information for searches run by all organizations. While the complexity of the search query can impact search times, the biggest factor that affects how long searches take is the number of mailboxes searched. Although Microsoft doesn't provide a Service Level Agreement for search times, the following table lists average search times for collection searches based on the number of mailboxes included in the search.

Reviewer: ```suggestion
We collect performance information for searches run by all organizations. While the complexity of the search query can impact search times, the biggest factor that affects how long searches take is the number of mailboxes searched. Although we don't provide a Service Level Agreement for search times, the following table lists average search times for collection searches based on the number of mailboxes included in the search.
```

> Number of mailboxes	Average search time

Reviewer: ```suggestion
| Number of mailboxes| Average search time |
|:-----|:-----|
|100  <br/> |30 seconds<br/> |
|1,000  <br/> |45 seconds<br/> |
|10,000  <br/> |4 minutes<br/> |
|25,000  <br/> |10 minures<br/> |
|50,000  <br/> |20 minutes<br/> |
|100,000  <br/> |25 minutes<br/> |
```

> 100

Reviewer: ```suggestion
```

> 30 seconds

Reviewer: ```suggestion
```

> 1,000

Reviewer: ```suggestion
```

> 45 seconds

Reviewer: ```suggestion
```

> 10,000

Reviewer: ```suggestion
```

> 4 minutes

Reviewer: ```suggestion
```

> 25,000

Reviewer: ```suggestion
```

> 10 minutes

Reviewer: ```suggestion
```

> 50,000

Reviewer: ```suggestion
```

> 20 minutes

Reviewer: ```suggestion
```

> 100,000

Reviewer: ```suggestion
```

> 25 minutes

Reviewer: ```suggestion
```
